src/navigation/orbitalnavigator.cpp

```diff
@@ -661,6 +661,19 @@ void OrbitalNavigator::updateStatesFromInput(const MouseInputState& mouseInputSt
     else {
         tickIdleBehaviorTimer(deltaTime);
     }
+
+    bool cameraLocationChanged = _mouseStates.hasNonZeroVelocities(true) ||
+        _joystickStates.hasNonZeroVelocities(true) ||
+        _websocketStates.hasNonZeroVelocities(true) ||
+        _scriptStates.hasNonZeroVelocities(true);
+
+    if (cameraLocationChanged && (_movementTimer < 0.0)) {
```

Comment:
```suggestion
    if (cameraLocationChanged && (_movementTimer < 0.f)) {
```

since `_movementTimer` is a `float`